src/store/backend/sagas.js

```diff
@@ -75,8 +76,12 @@ export function* setupBackend() {
     console.info('Reconnected')
   }
 
+  // Init APICLient
   yield call(() => apiClient.init())
 
+  // Fetch wallet claim + balance
+  yield all([put(getClaim()), put(getBalance())])
```

Comment:
can we do this on page load rather than on all sagas?

Reply:
discussed on call - makes sense to do it before page load. but this doesn't delay load or anything since it's just firing some actions